include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
> really bad idea: config.h is widely used header which is automatically generated by some autotools. as i can see config.h is not exported on 'make install', and it may conflict with user's configuration
> 
> or i missed something?

It doesn't need to be exported on `make install`, this `config.h` is created by autotools and one of the things it defines is whether it is a debug build or production build. The file is included at compile time. 

How would it conflict with a users configuration?